hooks/useAlternateObservationZones.ts

```diff
@@ -0,0 +1,104 @@
+import {Logger} from 'browser-bunyan';
+import {safeFetch} from 'hooks/fetch';
+import {LoggerContext, LoggerProps} from 'loggerContext';
+import {useContext} from 'react';
+import {AlternateObservationZones, KMLData, KMLPlacemark} from 'types/nationalAvalancheCenter';
+import {xml2json} from 'xml-js';
+
+export const useAlternateObservationZones = (url: string): UseQueryResult<AlternateObservationZones, AxiosError> => {
+  const {logger} = useContext<LoggerProps>(LoggerContext);
+  const key = queryKey(url);
+  const thisLogger = logger.child({query: key});
+
+  return useQuery<AlternateObservationZones, AxiosError>({
+    queryKey: key,
+    queryFn: (): Promise<AlternateObservationZones> => fetchAlternateObservationZones(thisLogger, url),
+    enabled: !!url,
+    cacheTime: 60, // TODO: Change cache to one day after testing
+    staleTime: 60,
+    initialData: [],
+  });
+};
+
+export const prefetchAlternateObservationZones = async (queryClient: QueryClient, url: string, logger: Logger) => {
+  const key = queryKey(url);
+  const thisLogger = logger.child({query: key});
+  thisLogger.debug('prefetching alternate observation zones');
+
+  await queryClient.prefetchQuery({
+    queryKey: key,
+    queryFn: async (): Promise<AlternateObservationZones> => {
+      thisLogger.trace(`prefetching`);
+      const result = fetchAlternateObservationZones(thisLogger, url);
+      return result;
+    },
+    cacheTime: 60, // TODO: Change cache to one day after testing
+    staleTime: 60,
+  });
+};
+
+export const fetchAlternateObservationZones = async (logger: LoggerProps['logger'], url: string): Promise<AlternateObservationZones> => {
+  const thisLogger = logger.child({url: url});
+  thisLogger.debug('Fetching alternate observation zones');
+  const response = await safeFetch(() => axios.get<string>(url), thisLogger, 'alternate observation zones');
+  try {
+    const alternateZones = parseKmlData(response);
+    thisLogger.debug('Fetched alternate observation zones');
+    return alternateZones;
+  } catch (error) {
+    thisLogger.error({error: error}, 'Error parsing KML data');
+    throw new Error('Error parsing KML data');
+  }
+};
+
+function queryKey(url: string) {
+  return ['alternateZoneKML-', url];
+}
+
+export function parseKmlData(response: string): AlternateObservationZones {
+  const kmlDataJson = xml2json(response, {compact: true, spaces: 2});
+  const kmlData = JSON.parse(kmlDataJson) as KMLData;
+  const placemarks = kmlData.kml.Document.Folder.Placemark;
+  const alternateZones: AlternateObservationZones = (Array.isArray(placemarks) ? placemarks : [placemarks]).map(placemark => {
+    let coordinates: number[][] = [];
+
+    const coordinateString = getCoordinateString(placemark);
+    coordinates = parseCoordinates(coordinateString);
+    const feature: AlternateObservationZones = {
+      type: 'Feature',
+      geometry: {
+        type: Array.isArray(coordinates[0][0]) ? 'MultiPolygon' : 'Polygon',
```

Comment:
I might be looking at an older version of this - did you decide if it would work to collapse this to `MultiPolygon` in every case?